### ISSUE TITLE
No of steps required to train COCO dataset on SSD Inception V2

### ISSUE THREAD

**Issue by jillelajitta** [tensorflow/models#5723]
Hi,
Could someone please tell me how many steps are required to train COCO dataset on SSD Inception V2 model?

Thanks.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by MaesIT** [comment]
I am assuming this question is concerning object detection.
If you used a pretrained model, none I guess.
If you want to train from scratch, i think in the config it is mentioned that 200k steps should be sufficient to get a decent result.

in the configs:
  ## Note: The below line limits the training process to 200K steps, which we
  ## empirically found to be sufficient enough to train the pets dataset. This
  ## effectively bypasses the learning rate schedule (the learning rate will
  ## never decay). Remove the below line to train indefinitely.

**Comment by karmel** [comment]
Closing, as this seems to be resolved.
